### PR TITLE
Allows loading configuration recursively

### PR DESCRIPTION
**Pull request by marcinc**
See commit message for details.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,9 +9,9 @@ ENV LS_VERSION 5.1.2
 RUN wget -q https://artifacts.elastic.co/downloads/logstash/logstash-${LS_VERSION}.tar.gz -O - | tar -xzf -; \
   mv logstash-${LS_VERSION} /logstash
 
-RUN /logstash/bin/logstash-plugin install --version 6.2.1 logstash-output-elasticsearch && \
-    /logstash/bin/logstash-plugin install --version 0.3.1 logstash-filter-kubernetes && \
-    /logstash/bin/logstash-plugin install --version 2.0.0 logstash-input-journald
+RUN JARS_SKIP=true /logstash/bin/logstash-plugin install --version 6.2.4 logstash-output-elasticsearch && \
```
Comment (vaijab) [inline Dockerfile:12]: Could you add a comment explaining why `JARS_SKIP=true` is needed and links to issues. Thanks!

**Review by vaijab — CHANGES_REQUESTED** [review]
One change, LGTM otherwise

File: Dockerfile (hunk 1 above)
Comment (marcinc) [reply Dockerfile:12]: It's in the commit already :)

**Review by vaijab — APPROVED** [review]
Perfecto!

**Pull request merged**